gradient/api_sdk/clients/experiment_client.py

```diff
@@ -362,9 +472,13 @@ def get(self, experiment_id):
     def logs(self, experiment_id, line=0, limit=10000):
         """Get list of logs for an experiment
 
-        :param str experiment_id:
+        *EXAMPLE*::
```

Comment:
could you add a description of what this does so that users can understand, since it's not necessarily intuitive? such as `Show the latest logs from the specified experiment.`